app/bundles/UserBundle/Resources/views/User/list.html.twig

```diff
@@ -119,6 +133,8 @@
                     <a href="mailto: {{ item.getEmail() }}">{{ item.getEmail() }}</a>
                 </td>
                 <td class="visible-md visible-lg">{{ item.getRole().getName() }}</td>
+                <td class="visible-md visible-lg">{{ dateToText(item.getLastLogin(),'local', 'Y-m-d H:i:s', true) }}</td>
+                <td class="visible-md visible-lg">{{ item.getUserStatus() }}</td>
```

Comment:
I like the PR and works as expected. 
But the status should be translateable.
You can add a translation key or you can use the same switch button before the name like we use in other entities (Email etc...)

https://github.com/mautic/mautic/blob/d71760b47dc9ba0add89e26099db025c4d2279b2/app/bundles/EmailBundle/Resources/views/Email/_list.html.twig#L116

Reply:
Hi! I added switch button before the name as You said, check it please